lib/resty/core/socket.lua

```diff
@@ -0,0 +1,124 @@
+local base = require "resty.core.base"
```

Comment:
You should handle the stream subsystem.

Reply:
Has handled here: 
https://github.com/openresty/lua-resty-core/blob/a53e3e76dd0f26746c205b5dde9892c7d0d045d5/lib/resty/core.lua#L22

Reply:
@poorsea That's not enough. This module should also have `base.allows_subsystem('http')`. See resty.core.ndk for an example.